docs/versioned/flink/flink-getting-started.md

```diff
@@ -547,6 +547,33 @@ RewriteDataFilesActionResult result = Actions.forTable(table)
 
 For more doc about options of the rewrite files action, please see [RewriteDataFilesAction](../../../javadoc/{{% icebergVersion %}}/org/apache/iceberg/flink/actions/RewriteDataFilesAction.html)
 
+## Configuration.
+
+The following options can be used to tune the performance of the query execution.
+
+| Key                                             | Default | Type    | Description                                                                                                                      |
+|:-----------------------------------------------:| ------- | ------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| table.exec.iceberg.infer-source-parallelism     | True    | Boolean | If is false, parallelism of source are set by config.<br/>If is true, source parallelism is inferred according to splits number. |
```

Comment:
Should we uppercase the first character for the default value `true` ?  I think both SQL job and Java DataStream job will use the constant `true` or false, right ?

Reply:
done.